helm/crds/services.k8s.aws_adoptedresources.yaml

```diff
@@ -183,12 +183,12 @@ spec:
                             name:
                               description: |-
                                 Name of the referent.
-                                More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names#names
+                                More info: http://kubernetes.io/docs/user-guide/identifiers#names
```

Comment:
I think you need to `git pull --rebase` the runtime repository

Reply:
done